rpc/auth.go

```diff
@@ -200,6 +200,60 @@ func MakeSimpleMultiAuthHandler(forEntities, expectedPayloads []string) AuthHand
 	})
 }
 
+// MakeSimpleMultiAuthPairHandler works similarly to MakeSimpleMultiAuthHandler with the addition of
+// supporting a key, id pair used to ensure that a key that maps to the id matches the key passed
+// during the function call.
+func MakeSimpleMultiAuthPairHandler(forEntities, expectedPayloads []map[string]string) AuthHandler {
```

Comment:
could you explain a bit to me about how we're going to be calling this? will `forEntities` be a list of robot FQDNs? and then the payload will be a map {keyId: key}? want to make sure i'm fully understanding the use case.

the reason why I ask is, for all the sdks, it seems like we're passing in they api key ID as the "entity" so am wondering how that would work with this structure.